test/test_functional.py

```diff
@@ -68,6 +77,8 @@ def _add_token(self, test_scripting):
         func = functional.add_token
         if test_scripting:
             func = torch.jit.script(func)
+
+        # List[List[int]]
```

Comment:
nit: can we prefix these comments with `case:`. At a first glance it almost looks like commented code.